src/e2eTests/java/uk/gov/hmcts/sptribs/e2e/CreateFlag.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.sptribs.e2e;
+
+
+import com.microsoft.playwright.Page;
+import com.microsoft.playwright.options.AriaRole;
+import com.microsoft.playwright.options.SelectOption;
+import com.netflix.hystrix.contrib.javanica.utils.FutureDecorator;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.sptribs.testutils.DateHelpers;
+import uk.gov.hmcts.sptribs.testutils.PageHelpers;
+
+import java.util.Calendar;
+
+import static com.microsoft.playwright.assertions.PlaywrightAssertions.assertThat;
+import static uk.gov.hmcts.sptribs.testutils.AssertionHelpers.textOptionsWithTimeout;
+import static uk.gov.hmcts.sptribs.testutils.PageHelpers.clickButton;
+import static uk.gov.hmcts.sptribs.testutils.PageHelpers.getTextBoxByLabel;
+
+public class CreateFlag extends Base {
+
+    @Test
+    public void caseWorkerShouldBeAbleToCreateACaseFlag(){
+        Login login = new Login();
+        login.loginAsStTest1User();
+        Case newCase = new Case();
+        newCase.createCase();
+        newCase.buildCase();
+        newCase.createCaseFlag();
+
```

Comment:
Trivial, but can you tidy up the whitespace?

Reply:
Resolved